File: Problem#9/Solution.js
/**
 * @param {number} x
 * @return {boolean}
 */
 var isPalindrome = function(x) {
    let number = x.toString();
    let numberReverse = stringReverse(number);
    return number == numberReverse;
};

function stringReverse(s) {
     return s.split('').reverse().join('');
}

File: Problem#9/Solution.cs
public class Solution {
    public bool IsPalindrome(int x) {
        string number = x.ToString();
        // creating a new instance of String with an array
        string numberReverse = new String(number.Reverse().ToArray());
        // return ture of number equals to numberReverse
        return number.Equals(numberReverse);
    }
}